### PR TITLE
[pt-br] Add docs/reference/node/_index.md

### PR DESCRIPTION
**Pull request by italofernandes13**
Translate docs to pt-br

docs/reference/node/_index.md

Related issues
https://github.com/kubernetes/website/issues/13939

### CODE REVIEW

File: content/pt-br/docs/reference/node/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Informações de referência do Node
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:2]: We should translate Node to "nó", as it is a common computing term in Portuguese.

File: content/pt-br/docs/reference/node/_index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+weight: 80
+no_list: true
+---
+
+Esta seção contém os seguintes tópicos de referência sobre os nodes:
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:7]: Same here.

File: content/pt-br/docs/reference/node/_index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+* O [checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) do kubelet
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:9]: How do you feel about using "A API de checkpoint do kubelet" instead?

"checkpoint API" is not very idiomatic in Portuguese.

File: content/pt-br/docs/reference/node/_index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+* Uma lista de [Artigos sobre a remoção do dockershim e sobre o uso de tempos de execução compatíveis com o CRI](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:11]: We have standardized the translation of "runtime" as "agente de execução" in Portuguese.

File: content/pt-br/docs/reference/node/_index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+Você também pode ler os detalhes de referência do node em outros lugares na
+documentação do Kubernetes, incluindo:
+
+* [Dados de métricas do Node](/docs/reference/instrumentation/node-metrics).
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:16]: We can translate node in these two sentences too.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @italofernandes13! 👋

Thank you very much for your contribution. 🎉 

Some small adjustments are needed in this page to follow some agreements and standards we have for the Brazilian Portuguese localization. If you have any questions or need assistance, please feel free to reach out to the Brazilian Portuguese localization team in the `#kubernetes-docs-pt` channel of Kubernetes Slack. 😃

File: content/pt-br/docs/reference/node/_index.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+no_list: true
+---
+
+Esta seção contém os seguintes tópicos de referência sobre os nós:
+
+* A [API de checkpoint](/docs/reference/node/kubelet-checkpoint-api/) do kubelet
+
+* Uma lista de [Artigos sobre a remoção do dockershim e sobre o uso de agentes de execução compatíveis com o CRI](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)
+
+Você também pode ler os detalhes de referência do node em outros lugares na
```
Comment (stormqueen1990) [inline content/pt-br/docs/reference/node/_index.md:13]: One leftover "node" here.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @italofernandes13! 👋

Thanks for addressing the previous feedback! 😃

I found one "node" reference leftover. Other than that, it looks good to me!

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

**Comment by stormqueen1990** [comment]
Hi there, @italofernandes13! 👋

This PR is a LGTM, but could you please take another look at your rebase commits? Commit 0df498edc839acc4dcf96fe077209b070df99d90 is looking a bit strange (it appears to be some kind of a squashed rebase and I think it should not be in this branch).

Thanks in advance!

**Comment by stormqueen1990** [comment]
/cc @devlware
Could you please provide a second review?

/hold for checking the rebase commit.

File: content/pt-br/docs/reference/node/_index.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+Esta seção contém os seguintes tópicos de referência sobre os nós:
+
+* A [API de checkpoint](/docs/reference/node/kubelet-checkpoint-api/) do kubelet
+
+* Uma lista de [Artigos sobre a remoção do dockershim e sobre o uso de agentes de execução compatíveis com o CRI](/docs/reference/node/topics-on-dockershim-and-cri-compatible-runtimes/)
+
+Você também pode ler os detalhes de referência do nó em outros lugares na
+documentação do Kubernetes, incluindo:
+
+* [Dados de métricas do Nó](/docs/reference/instrumentation/node-metrics).
```
Comment (devlware) [inline content/pt-br/docs/reference/node/_index.md:16]: There was an update to this doc so I`m suggesting the new link below.

```suggestion
* [Dados de métricas do Nó](/docs/reference/instrumentation/node-metrics).
* [Métricas de Contêiner & Pod CRI](docs/reference/instrumentation/cri-pod-container-metrics).
```

**Review by devlware — COMMENTED** [review]
@italofernandes13 Thank you for you contribution and sorry for the late reply. I probably lost this notification in my inbox. :(

**Comment by rikatz** [comment]
This needs rebase :)

**Comment by devlware** [comment]
Hi @italofernandes13 can you please rebase ?

**Comment by stormqueen1990** [comment]
Hello there, @italofernandes13! 👋

Thank you very much for your work on this PR. However, it has been open and awaiting rebase for a while with no action taken. For this reason, I will be closing this PR. If you wish to continue your work on this, please feel free to reopen it.

Should you have any questions, please don't hesitate to reach out to the `#kubernetes-docs-pt` channel in Kubernetes Slack.

---

Olá, @italofernandes13! 👋

Obrigada pelo teu trabalho neste PR. Devido a inatividade após a solicitação de rebase, irei fechá-lo. No entanto, caso desejes continuar teu trabalho, podes reabrir o PR a qualquer momento.

Se tiveres dúvidas, não hesita em contatar-nos através do canal `#kubernetes-docs-pt` no Slack do Kubernetes.

/close

**Pull request closed without merging**